src/app/camo-card/camo-card.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-camo-card',
+  templateUrl: './camo-card.component.html',
+  styleUrls: ['./camo-card.component.scss']
+})
+export class CamoCardComponent implements OnInit {
+
```

Comment:
this now needs something like 

```ts
@Input() weapon: Weapon;
@Input() progress: Progress;
```

On the home-page.component.ts, move the `interface Progress` to a new file and have it exported: `export interface Progress`. This way this definition can be shared between homepage and CamoCardComponent.

Reply:
You'll then have to import this interface in both home-page.component.ts and camo-card-component.ts. If you do control + '.' on where the error squiggly line appears you can trigger hints.

Reply:
exported the interface progress to a new file